friendly-frogs/friendly_frogs/morning/apis_call.py

```diff
@@ -0,0 +1,20 @@
+from newsapi.newsapi_client import NewsApiClient
+
+NEWS_SOURCES = (
+    ('bbc-news', 'BBC NEWS'),
+)
```

Comment:
Is this constant supposed to be used in the `get_news` method instead of the parameter there or what is its use?